### PR TITLE
Fix mutation tests

### PR DESCRIPTION
**Pull request by ielis**
@justaddcoffee I am proposing changes to the mutation tests to make them pass.

The issue with the code was that the functions `test_gene` and `test_hgvs` did not include all required fields. I changed the test inputs such that the info is present and the tests only update the tested fields, such as `Hugo_Symbol` and `HGVSc`.

Some tests fail even after this PR, but we need to resolve those separately.

---

Pls let me know if the changes correspond to what you intended to test back in the September..

### CODE REVIEW

File: src/oncoexporter/cda/cda_factory.py [hunk 1]
```diff
@@ -3,17 +3,13 @@
...
 import pandas as pd
-from .cda_downloader import CdaDownloader
+
 
 class CdaFactory(metaclass=abc.ABCMeta):
     """Superclass for the CDA Factory Classes
 
     Each subclass must implement the to_ga4gh method, which transforms a row of a table from CDA to a GA4GH Message.
     """
-    def __init__(self, overwrite_downloads:bool=False):
```
Comment (justaddcoffee) [inline src/oncoexporter/cda/cda_factory.py:13]: I'm guessing you removed this b/c downloading isn't actually used here? Just curious

**Review by justaddcoffee — APPROVED** [review]

File: src/oncoexporter/cda/cda_factory.py (hunk 1 above)
Comment (ielis) [reply src/oncoexporter/cda/cda_factory.py:13]: Adding the download functionality here implies that *every* `CdaFactory` instance needs the download functionality and I do not believe that is the case. What `CdaFactory` (may) need is a way to map ICD-O to NCIT, nothing else. The mapping code should be completely indifferent of the implementation, e.g. if you get the mapping from remote server, from a `dict`, file, database, whatever.

However, here the `__init__` *must* use the `Downloader`, know about some `overwrite_downloads` parameter, execute some arbitrary code (`get_icdo_to_ncit_path()`). All this is unrelated to the functionality, which is *take a row and give me something*.

So it needs to be moved elsewhere, probably to the place where it is used.

If we find ourselves needing to map ICD-O to NCIT at several places, we should create a class for that.

Comment (justaddcoffee) [reply src/oncoexporter/cda/cda_factory.py:13]: Thanks Daniel that makes sense, thanks for clarifying

Comment (ielis) [reply src/oncoexporter/cda/cda_factory.py:13]: The extent to which we want to apply these code Nazi rules is another question.. :grimacing: :smile_cat:

**Pull request merged**